amplify/backend/auth/covidcounterba0a32fb/parameters.json

```diff
@@ -45,16 +45,12 @@
             "Arn"
         ]
     },
-    "useDefault": "defaultSocial",
+    "useDefault": "default",
     "usernameAttributes": [
         "email"
     ],
     "userPoolGroupList": [],
     "serviceName": "Cognito",
     "dependsOn": [],
-    "hostedUI": true,
```

Comment:
This should not have been removed

Reply:
Hey Shannon this code is present currently , comma has been removed by new changes